ci-operator/step-registry/openshift/e2e/aws/single-node/graceful-shutdown/openshift-e2e-aws-single-node-graceful-shutdown-workflow.yaml

```diff
@@ -0,0 +1,15 @@
+workflow:
+  as: openshift-e2e-aws-single-node-graceful-shutdown
+  steps:
+    pre:
+    - chain: ipi-conf-aws
+    - ref: single-node-conf-aws-graceful-shutdown
```

Comment:
Any reason not to reuse existing steps, like this?:
```yaml
    - chain: ipi-conf-aws
    - ref: single-node-conf-aws
    - ref: single-node-conf-e2e
    - ref: single-node-conf-aws-graceful-shutdown
    - chain: ipi-install
```

Then either:
- have the graceful-shutdown only modify the existing `${SHARED_DIR}/manifest_single-node-reserve-sys-mem-kubeletconfig.yml` by appending the graceful shutdown stanzas. A bit weird because the name of the file would then be a bit misleading
- have the graceful-shutdown create another KubeletConfig manifest. Does the MCO support multiple KubeletConfigs by merging? If yes, then this option would be possible and ideal.
- Keep `single-node-conf-aws` but remove `single-node-conf-e2e` and replace it with single-node-conf-graceful-shutdown` (note that I removed aws because then it wouldn't need to be aws specific)

Reply:
There's still `aws` here